desktop/apps/artsy_primer/routes.js

```diff
@@ -19,3 +22,22 @@ export const personalize = async (req, res) => {
   }
   res.render('personalize')
 }
+
+export const setSailthruData = async (req, res) => {
```

Comment:
Right and https://github.com/artsy/force/pull/1073/files#diff-f6777e97912484892bb42cd8515b5023R113